pkg/pool-stable-phantom/test/StablePhantomPool.test.ts

```diff
@@ -1141,23 +1326,33 @@ describe('StablePhantomPool', () => {
           i == bptIndex ? bn(0) : fp(100)
         );
         await pool.init({ recipient: sender, initialBalances: equalBalances });
+
+        const result = await pool.getTokens();
+        allTokens = result.tokens;
       });
 
-      context('when not paused', () => {
-        it('cannot exit proportionally', async () => {
-          const bptIn = fp(10);
-          await expect(pool.proportionalExit({ from: lp, bptIn })).to.be.revertedWith('NOT_PAUSED');
+      context('when not in recovery mode', () => {
+        it('reverts', async () => {
+          const totalBptBalance = await pool.balanceOf(lp);
+
+          await expect(
+            pool.recoveryModeExit({
+              from: lp,
+              tokens: allTokens,
+              currentBalances: initialBalances,
+              bptIn: totalBptBalance,
+            })
+          ).to.be.revertedWith('NOT_IN_RECOVERY_MODE');
         });
       });
 
-      // Skip until recovery mode implemented
-      context.skip('when paused', () => {
-        context('one lp', () => {
-          sharedBeforeEach('pause pool', async () => {
-            await pool.pause();
-          });
+      context('when in recovery mode', () => {
+        sharedBeforeEach('enable recovery mode (and simulate utter failure)', async () => {
```

Comment:
I'm not seeing how we're simulating failure here. Surely that's happening on L1446?

Reply:
That got left over after shuffling things around. Will remove; 1446 covers that case.